django-cloudlaunch/baselaunch/domain_model.py

```diff
@@ -55,5 +55,16 @@ def get_cloud_provider(cloud, cred_dict):
         config.update(cred_dict)
         return CloudProviderFactory().create_provider(ProviderList.AZURE,
                                                       config)
+    elif isinstance(cloud, models.GCE):
+        config = {
+            'gce_client_email': cred_dict['client_email'],
```

Comment:
@machristie Looking at this, it looks like gce_client_email and gce_project_name are stored redundantly in gce_service_creds_dict. Should we just change the gce provider interface to only use the gce_service_creds_dict?